pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -68,12 +72,15 @@ type VolumeBinding struct {
 	Binder                               scheduling.SchedulerVolumeBinder
 	PVCLister                            corelisters.PersistentVolumeClaimLister
 	GenericEphemeralVolumeFeatureEnabled bool
+	scorer                               *volumeCapacityScorer
```

Comment:
if we make volumeCapacityScorer a pure function type, here we can define it `score volumeCapacityScorer `

Reply:
updated